Makefile

```diff
@@ -52,7 +52,7 @@ lint: $(BIN)/golangci-lint $(BIN)/buf ## Lint Go and protobuf
 
 .PHONY: lintfix
 lintfix: $(BIN)/golangci-lint $(BIN)/buf ## Automatically fix some lint errors
-	golangci-lint run --fix
+	golangci-lint run --fix --modules-download-mode=readonly --timeout=3m0s
```

Comment:
@oliversun9, is this just for consistency with other repos somewhere, or was this an issue in CI?

Also, I suspect you meant to add these flags to the invocation up above, on line 49, instead of here.

Reply:
Consistency with other repos. And yes, agree that these flags should probably go on both this invocation and the one above.

Reply:
> I suspect you meant to add these flags to the invocation up above, on line 49, instead of here.

Great catch! Yes, I meant to add it to `lint`. Now it's added to both invocations.